core/src/main/resources/error/error-classes.json

```diff
@@ -96,10 +144,21 @@
     "message" : [ "Unrecognized SQL type %s" ],
     "sqlState" : "42000"
   },
+  "UNSUPPORTED_CHANGE_COLUMN" : {
+    "message" : [ "Please add an implementation for a column change here" ],
+    "sqlState" : "0A000"
+  },
+  "UNSUPPORTED_DATATYPE" : {
+    "message" : [ "Unsupported data type %s %s" ],
```

Comment:
This is a little awkward in the case that there is no value for the first param; as discussed above, I think we should combine the two-param case into one string param.